### ISSUE TITLE
ESP32-S3 wireless stick V3 433

### ISSUE THREAD

**Issue by panban-ux** [G4lile0/tinyGS#230]
As a new to TinyGs, im seeing that not an implementation has been made through this device.
My question is, would a TinyGS implementation adds coverage to a terrestrial application when not TTN/Lorawan gateways in the traditional way are around?
For that one must assume that with this infrastructure we can only listen to Lora satellites and not sent an uplink from a remote location the way traditional nodes do?

In short thats a ground gateway station TinyGs do?

Also my device isnt supported and it can it be implemented through templates?
